stories/core/designGrid.css

```diff
@@ -55,7 +56,7 @@
     background-image: linear-gradient(
       90deg,
       transparent var(--col),
-      color(var(--blueBerry) alpha(0.1)) var(--gutter)
+      color(var(--bolt) alpha(0.1)) var(--gutter)
```

Comment:
I think the color palette is unlikely to change ever on any project unless you are creating something like bootstrap. It's also easy to regexp.

Colors here have no meaning other than showing a bunch of different blocks. I could see naming them something like `color1`, `color2`, `color3` to emphasize their lack of meaning but that's not much different from what we have now and it makes it easier to read the css